data/core/keymap.lua

```diff
@@ -35,24 +40,52 @@ local modkey_map = modkeys_os.map
 local modkeys = modkeys_os.keys
 
 
+local stroke_sep = " "
```

Comment:
> the current stroke separator is a space ( ), which obviously doesn't play nice with the keypad keys

What do you think about using `,`?

Reply:
Mmm, I wonder if we should just clean up the key names either in https://github.com/lite-xl/lite-xl/blob/1a5fd97c20438a5035c0e018754ebb6abd80b7db/src/api/system.c#L118 or somewhere on the Lua side, to replace spaces with `_`.
It would be a breaking change, but I doubt many people are using keypad keymaps.